src/test/ensemble_tests.cpp

```diff
@@ -664,7 +722,12 @@ TEST_F(EnsembleFlowTest, FailInDLNodeFetchResultsStreamIdReleasedForDeferredNode
     auto input_node = std::make_unique<EntryNode>(&request, inputsInfo);
     auto failInFetchNode = std::make_unique<DLNodeFailInFetch>("failInFetch_node", dummyModelName, requestedModelVersion, managerWithDummyModel);
     auto modelNode = std::make_unique<DLNodeFailInFetch>("dummy_node", dummyModelName, requestedModelVersion, managerWithDummyModel);
-    auto output_node = std::make_unique<ExitNode>(&response);
+    auto tensorInfo = std::make_shared<ovms::TensorInfo>(customPipelineOutputName,
```

Comment:
Maybe we could separate this tensorInfo to test_utils or fixture field? This way we would not need to initialize it in each test.